web/src/lib/components/shared-components/side-bar/side-bar-link.svelte

```diff
@@ -34,49 +38,71 @@
-    <Icon path={icon} size="1.5em" class="shrink-0" flipped={flippedLogo} ariaHidden />
-    <span class="text-sm font-medium">{title}</span>
-  </div>
-
-  <div
-    class="h-0 overflow-hidden transition-[height] delay-1000 duration-100 sm:group-hover:h-auto group-hover:sm:overflow-visible md:h-auto md:overflow-visible"
   >
-    {#if moreInformation}
-      <!-- svelte-ignore a11y_no_static_element_interactions -->
-      <div
-        class="relative flex cursor-default select-none justify-center"
-        onmouseenter={() => (showMoreInformation = true)}
-        onmouseleave={() => (showMoreInformation = false)}
-      >
-        <div class="p-1 text-gray-600 hover:cursor-help dark:text-gray-400">
-          <Icon path={mdiInformationOutline} />
-        </div>
+    <div class="flex w-full place-items-center gap-4 overflow-hidden truncate">
+      <Icon path={icon} size="1.5em" class="shrink-0" flipped={flippedLogo} ariaHidden />
+      <span class="text-sm font-medium">{title}</span>
+    </div>
 
-        {#if showMoreInformation}
-          <div class="absolute right-6 top-0">
-            <div
-              class="flex place-content-center place-items-center whitespace-nowrap rounded-3xl border bg-immich-bg px-6 py-3 text-xs text-immich-fg shadow-lg dark:border-immich-dark-gray dark:bg-gray-600 dark:text-immich-dark-fg"
-              class:hidden={!showMoreInformation}
-              transition:fade={{ duration: 200 }}
-            >
-              {@render moreInformation?.()}
-            </div>
+    <div
+      class="h-0 overflow-hidden transition-[height] delay-1000 duration-100 sm:group-hover:h-auto group-hover:sm:overflow-visible md:h-auto md:overflow-visible"
+    >
+      {#if moreInformation}
+        <!-- svelte-ignore a11y_no_static_element_interactions -->
+        <div
+          class="relative flex cursor-default select-none justify-center"
+          onmouseenter={() => (showMoreInformation = true)}
+          onmouseleave={() => (showMoreInformation = false)}
+        >
+          <div class="p-1 text-gray-600 hover:cursor-help dark:text-gray-400">
+            <Icon path={mdiInformationOutline} />
           </div>
-        {/if}
-      </div>
-    {/if}
-  </div>
-</a>
+
+          {#if showMoreInformation}
+            <div class="absolute right-6 top-0">
+              <div
+                class="flex place-content-center place-items-center whitespace-nowrap rounded-3xl border bg-immich-bg px-6 py-3 text-xs text-immich-fg shadow-lg dark:border-immich-dark-gray dark:bg-gray-600 dark:text-immich-dark-fg"
+                class:hidden={!showMoreInformation}
+                transition:fade={{ duration: 200 }}
+              >
+                {@render moreInformation?.()}
+              </div>
+            </div>
+          {/if}
+        </div>
+      {/if}
+    </div>
+  </a>
+  {#if hasDropdown}
+    <span class="hidden md:block absolute top-4 left-1">
+      <button
```

Comment:
- This button doesn't display correctly in firefox
- It should have an accessible name
- Maybe increase the clickable area, at least vertically